src/js/components/FileInput/FileInput.js

```diff
@@ -312,10 +368,15 @@ const FileInput = forwardRef(
                   weight={
                     theme.global.input.weight || theme.global.input.font.weight
                   }
-                  truncate
                   {...theme.fileInput.label}
+                  alignSelf="center"
                 >
-                  {file.name}
+                  <Box direction="row">
+                    {maxSize && file.size > maxSize ? <CircleAlert /> : null}
+                    <Text truncate margin={{ left: 'xsmall' }}>
```

Comment:
Ahh gotcha, yes in this code chunk it's definitely still up in the air since I will be implementing something like this once a submit happens instead of just having a conditional render here. But I will get this implemented for now with `gap="xsmall"` thanks for the heads up 👍